.github/workflows/test-command.yml

```diff
@@ -1,3 +1,4 @@
+# TODO deprecate
```

Comment:
If/when we deprecate this, change it to just respond with "This command has been deprecated" See [docs](https://docs.cloudposse.com/community/automated-testing/#chatops-usage).